### PR TITLE
K8s apply check mode fix

### PR DESCRIPTION
**Pull request by willthames**
##### SUMMARY
Adds test for applying a Service change in check_mode and ensures that test passes

Fixes ansible/ansible#66780

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
k8s/raw

##### ADDITIONAL INFORMATION
Needs backporting to ansible/ansible for 2.9

### CODE REVIEW

**Comment by willthames** [comment]
There is an intermittent test failure (there is a race condition) that is fixed by #87

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ provisioner:
         ansible_python_interpreter: '{{ ansible_playbook_python }}'
   env:
     ANSIBLE_FORCE_COLOR: 'true'
+  options:
+    vvv: True
```
Comment (geerlingguy) [inline molecule/default/molecule.yml:25]: I usually use default verbosity on test runs (when not specifically debugging an issue), but that's just me. @willthames and @fabianvf especially—do you prefer having more verbosity? I can go either way.

Comment (willthames) [reply molecule/default/molecule.yml:25]: Definitely prefer more verbosity - lets me validate far more assumptions (which python is in use, which module is actually being exercised) and shows the full invocation, diff and results in the output.

Comment (fabianvf) [reply molecule/default/molecule.yml:25]: I prefer greater verbosity as well, annoying to have to rerun a failed build by hand when the cause of an error is unclear

**Pull request merged**